pkg/controller/volume/cache/actual_state_of_world.go

```diff
@@ -132,7 +134,7 @@ type actualStateOfWorld struct {
 // believes to be succesfully attached to a node it is managing.
 type attachedVolume struct {
```

Comment:
The uber PR I have pending for the mount/unmount refactor moves the exported types into a separate file.